YourSpace/account/forms.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.auth.models import User
+from .models import Profile
+
+
+class LogInForm(forms.Form):
+    """
+        Form for logging in
+    """
```

Comment:
The comment in line 7-9 seems redundant because the class name is already clear enough to understand what's happening inside this particular class. There are 2 options for this:

1. Delete the comment.
2. Replace the the comment with something more informative just like how you implemented the code.

Remember to apply the changes to the rest of your codes as well to make it uniform. Thanks.